client/packages/android/README.MD

```diff
@@ -0,0 +1,103 @@
+# Mobile App with Discovery
+
+The client app is bundled with the server, on Android app can run as standalone site with a local server, as a client connecting to another server or as a server for other clients.
```

Comment:
```suggestion
The Android app can run as standalone site with a local server, as a client connecting to another server or as a server for other clients.
```
First part is already mentioned in the last sentence

Reply:
🙏